### DISCUSSION TITLE
HISTARFM issues

### DISCUSSION THREAD

**Discussion by samapriya** [samapriya/awesome-gee-community-datasets#189] [General]
Creating a place for users to ask questions or request help as needed

**Comment by Super184666** [comment]
Could you please provide the code of HISTARFM? Thanks a lot.

**Reply by samapriya** [reply]
Please reach out to the original authors [find the paper here](https://www.sciencedirect.com/science/article/pii/S0034425720302716)

**Reply by Super184666** [reply]
Thank you.
